build.gradle

```diff
@@ -365,19 +365,6 @@ subprojects {
   test.dependsOn('checkstyleMain', 'checkstyleTest')
 
   if (!JavaVersion.current().isJava9Compatible()) {
-    findbugs {
-      toolVersion = "3.0.1"
-      excludeFilter = file("$rootDir/gradle/findbugs-exclude.xml")
-      ignoreFailures = false
-    }
-    test.dependsOn('findbugsMain')
-
-    tasks.withType(FindBugs) {
-      reports {
-        xml.enabled(project.hasProperty('xmlFindBugsReport'))
-        html.enabled(!project.hasProperty('xmlFindBugsReport'))
-      }
-    }
```

Comment:
Could you please explain what this change is for.

Reply:
Thanks for noticing this. I added this back. I made this change when I am testing the patch locally  because otherwise it is unnecessarily slow to run findBugs. This change is not intended to be committed.